x/lockup/types/msgs.go

```diff
@@ -43,11 +63,20 @@ func (m MsgBeginUnlocking) GetSignBytes() []byte {
 	return sdk.MustSortJSON(ModuleCdc.MustMarshalJSON(&m))
 }
 func (m MsgBeginUnlocking) GetSigners() []sdk.AccAddress {
-	return []sdk.AccAddress{m.Owner}
+	owner, _ := sdk.AccAddressFromBech32(m.Owner)
+	return []sdk.AccAddress{owner}
 }
 
 var _ sdk.Msg = &MsgUnlockTokens{}
 
+// NewMsgUnlockTokens creates a message to begin unlocking all tokens of a user
+//nolint:interfacer
```

Comment:
Isn't interfacer deprecated (https://github.com/mvdan/interfacer), it shouldn't be getting included in our lint list.